corehq/apps/userreports/tests/test_pillow.py

```diff
@@ -97,29 +99,34 @@ def setUp(self):
         with trap_extra_setup(KafkaUnavailableError):
             self.pillow.get_change_feed().get_current_offsets()
 
+    @run_with_all_backends
     def test_stale_rebuild(self):
         later_config = copy(self.config)
         later_config.save()
         self.assertNotEqual(self.config._rev, later_config._rev)
         with self.assertRaises(StaleRebuildError):
-            self.pillow.rebuild_table(IndicatorSqlAdapter(self.config))
+            self.pillow.rebuild_table(get_indicator_adapter(self.config))
 
     @patch('corehq.apps.userreports.specs.datetime')
+    @run_with_all_backends
     def test_change_transport(self, datetime_mock):
         datetime_mock.utcnow.return_value = self.fake_time_now
         sample_doc, expected_indicators = get_sample_doc_and_indicators(self.fake_time_now)
         self.pillow.process_change(doc_to_change(sample_doc))
         self._check_sample_doc_state(expected_indicators)
 
     @patch('corehq.apps.userreports.specs.datetime')
+    @run_with_all_backends
     def test_rebuild_indicators(self, datetime_mock):
         datetime_mock.utcnow.return_value = self.fake_time_now
         self.config.save()
         sample_doc, expected_indicators = get_sample_doc_and_indicators(self.fake_time_now)
         CommCareCase.get_db().save_doc(sample_doc)
```

Comment:
I see this is a different `run_with_all_backends` - I think you should rename it to `@run_with_all_ucr_backends` to avoid confusion